### PR TITLE
Feature: APP-2102 - Proposal action cards

### PR DESCRIPTION
**Pull request by RakeshUP**
## Description

- Action builder card and Execution widget card are completed for the New proposal page
- Execution widget on Proposal page is done(Decoding done using ConsenSys abi-decoder)
- Wired up all the form logic
- SCC modal state persists till the modal is closed. Switching contracts, functions won't affect the state

Task: [APP-2102](https://aragonassociation.atlassian.net/browse/APP-2102), [APP-2185](https://aragonassociation.atlassian.net/browse/APP-2185), [APP-2189](https://aragonassociation.atlassian.net/browse/APP-2189)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2102]: https://aragonassociation.atlassian.net/browse/APP-2102?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[APP-2185]: https://aragonassociation.atlassian.net/browse/APP-2185?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[APP-2189]: https://aragonassociation.atlassian.net/browse/APP-2189?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/containers/smartContractComposer/components/inputForm.tsx [hunk 1]
```diff
@@ -92,6 +154,7 @@ const ComponentForType: React.FC<ComponentForTypeProps> = ({
               }}
               placeholder="0x"
               adornmentText={value ? t('labels.copy') : t('labels.paste')}
+              disabled={disabled}
```
Comment (sepehr2github) [inline packages/web-app/src/containers/smartContractComposer/components/inputForm.tsx]: use disabledFilled here to keep the adornment button active please

File: packages/web-app/src/containers/smartContractComposer/components/inputForm.tsx [hunk 2]
```diff
@@ -111,8 +174,7 @@ const ComponentForType: React.FC<ComponentForTypeProps> = ({
       return (
         <Controller
           defaultValue=""
-          name={`sccActions.${functionName}.${input.name}`}
-          control={control}
+          name={formName}
```
Comment (sepehr2github) [inline packages/web-app/src/containers/smartContractComposer/components/inputForm.tsx:187]: Thanks, quick question, How do you manage the mandotary fields?

Comment (RakeshUP) [reply packages/web-app/src/containers/smartContractComposer/components/inputForm.tsx:187]: There's no validation as of now. Validation needs to be handled as a separate ticket. I think it's a subtask in APP-2182

**Review by sepehr2github — APPROVED** [review]
Thanks for the changes!!!

**Pull request merged**